server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/WriteLoadConstraintDeciderIT.java

```diff
@@ -604,6 +604,7 @@ private Settings enabledWriteLoadDeciderSettings(int utilizationThresholdPercent
                 WriteLoadConstraintSettings.WRITE_LOAD_DECIDER_QUEUE_LATENCY_THRESHOLD_SETTING.getKey(),
                 TimeValue.timeValueMillis(queueLatencyThresholdMillis)
             )
+            .put("cluster.routing.allocation.index_balance_decider.excess_shards", 1000)
```

Comment:
We should be able to remove this now that the decider is disabled by default right?